lib/routes/Play/PlayRoute.tsx

```diff
@@ -63,8 +64,14 @@ export function useLiveObject<T>(props: {
       const isSubscriber = !props.isOwner;
       const object = liveObject?.toObject();
       const objectKeys = Object.keys(object ?? {});
-      if (isSubscriber && object && objectKeys.length > 0) {
-        props.onChange(object as T);
+      if (props.canBeEmpty) {
+        if (isSubscriber && object) {
+          props.onChange(object as T);
+        }
+      } else {
+        if (isSubscriber && object && objectKeys.length > 0) {
+          props.onChange(object as T);
+        }
```

Comment:
Liveblocks initializes objects with `{}`, the issue is that it can be problematic if the code expects an object to be defined.

I've added a new `canBeEmpty`.

For the `session`, which always has `session.gm` or `session.players`, it cannot be empty. Otherwise the code would break.
For the `characters`, because it's a dictionnary, it CAN be empty and when it is it should still sync and call the `onChange`. Otherwise, if there's only one character and it's being removed, the sync will not happen